pegasus/sites.v3/code.org/public/private/k5survey/index.haml

```diff
@@ -22,119 +19,388 @@
+        #{all_surveys_totals[:responses].round}
+    %tr{style:'background-color:lightgreen'}
+      %td
+        Overall, rate your satisfaction about the workshop.
+      %td
+        #{(individual_survey_totals[:satisfaction_total ]/ individual_survey_totals[:responses]).round(2)}
+      %td
+      %td
+        #{(all_surveys_totals[:satisfaction_total] / all_surveys_totals[:responses]).round(2)}
+    %tr{style:'background-color:deepskyblue'}
+      %td
+        \% In-service teachers
+      %td
+        #{(individual_survey_totals[:number_in_service_teachers] / individual_survey_totals[:responses] * 100).round}%
+      %td
+      %td
+        #{(all_surveys_totals[:number_in_service_teachers] / all_surveys_totals[:responses] * 100).round}%
+    %tr{style:'background-color:deepskyblue'}
+      %td
+        \% in-school - vs afterschool/homeschool
+      %td
+        #{(individual_survey_totals[:number_in_school] / individual_survey_totals[:responses] * 100).round}%
+      %td
+      %td
+        #{(all_surveys_totals[:number_in_school] / all_surveys_totals[:responses] * 100).round}%
+    %tr{style:'background-color:deepskyblue'}
+      %td
+        \% who plan to offer the course
+      %td
+        #{(individual_survey_totals[:number_offering_again] / individual_survey_totals[:responses] * 100).round}%
+      %td
+      %td
+        #{(all_surveys_totals[:number_offering_again] / all_surveys_totals[:responses] * 100).round}%
+    %tr{style:'background-color:bisque'}
+      %td
+        My facilitator came prepared to teach/lead the workshop
+      %td
+        #{(individual_survey_totals[:facilitator_prepared_total] / individual_survey_totals[:responses] + 3).round(2)}
+      %td
+      %td
+        #{(all_surveys_totals[:facilitator_prepared_total] / all_surveys_totals[:responses] + 3).round(2)}
+    %tr{style:'background-color:bisque'}
+      %td
+        My facilitator was very knowledgeable in the workshop content and Code.org curriculum.
+      %td
+        #{(individual_survey_totals[:facilitator_knowledgeable_total] / individual_survey_totals[:responses] + 3).round(2)}
+      %td
+      %td
+        #{(all_surveys_totals[:facilitator_knowledgeable_total] / all_surveys_totals[:responses] + 3).round(2)}
+    %tr{style:'background-color:bisque'}
+      %td
+        My facilitator was an enjoyable person.
+      %td
+        #{(individual_survey_totals[:facilitator_enjoyable_total] / individual_survey_totals[:responses] + 3).round(2)}
+      %td
+      %td
+        #{(all_surveys_totals[:facilitator_enjoyable_total] / all_surveys_totals[:responses] + 3).round(2)}
+    %tr{style:'background-color:bisque'}
+      %td
+        My facilitator acted in a professional manner.
+      %td
+        #{(individual_survey_totals[:facilitator_professional_total] / individual_survey_totals[:responses] + 3).round(2)}
+      %td
+      %td
+        #{(all_surveys_totals[:facilitator_professional_total] / all_surveys_totals[:responses] + 3).round(2)}
```

Comment:
The 3 is from the g-sheet. I believe it is to normalize most fields on a 5 point scale.